config/prte_setup_pmix.m4

```diff
@@ -73,6 +73,7 @@ AC_DEFUN([PRTE_CHECK_PMIX],[
            AC_MSG_ERROR([Cannot continue.])])
 
     prte_external_pmix_save_CPPFLAGS=$CPPFLAGS
+    PRTE_FLAGS_PREPEND_UNIQ(CPPFLAGS, $prte_pmix_CPPFLAGS/pmix)
```

Comment:
This won't work in the general case.  $prte_pmix_CPPFLAGS is likely to include more than one -I (for example, if it used pkg-config or pmixcc, it should contain `-I${prefix}/include -I${prefix}/include/pmix`).  I thought I covered all the use cases such that you wouldn't need this.  Can you open an issue with the output of configure (or config.log) from PRRTE and I'll sort out what went wrong?

Reply:
Okay - I'm puzzled how this made it thru our CI as all PRs are now failing. Very odd.

Reply:
Oh now this is really weird - it built correctly this time! I'll poke some more at this to see what's going on.